Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use the latest Node image
-FROM node:latest
+FROM kkarczmarczyk/node-yarn
```

Comment:
I don't think we should switch to this image. It is out of date and doesn't seem to be in as wide of use as the `node` image itself; we need to make sure we are getting updates as quickly as possible in case there are security issues.